tests/ci/docker_images/linux-x86/ubuntu-20.04_gcc-7x_corretto/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu-20.04:accp_base
+
```

Comment:
what will adding an `aarch64` dimension to our CI look like? will we need parallel Dockerfile + images? is there some way to consolidate common configuration in a shared "base" dockerfile across multiple processor architectures?